tsconfig.json

```diff
@@ -22,21 +16,14 @@
     "rootDir": "src",
     "target": "es2015",
     "paths": {
-      "Assets/*": [
-        "./Assets/*"
-      ]
+      "Assets/*": ["./Assets/*"]
```

Comment:
One thing we should try to do if possible is use a root directory similar to [here](https://github.com/artsy/reaction/blob/master/.babelrc#L28) so that we don't have to define individual keys.